builder/builder.go

```diff
@@ -238,6 +238,8 @@ func (b *Builder) InitMix(upstreamVer string, mixVer string, allLocal bool, allU
 		upstreamVer = ver
 	}
 
+	fmt.Printf("Initializing mix version %s from upstream version %s\n", mixVer, upstreamVer)
```

Comment:
Any reason this is down here instead of at the top of the function?

Reply:
Wanted to print the actual version instead of latest.

Reply:
:+1: